experiment/simulation/js/main.js

```diff
@@ -1 +1,258 @@
-//Your JavaScript goes in here
+let detergentAddCounter = 1;
+
+// disable onclick of all components except of spoon and detergent
+if(document.getElementsByClassName("clothWithStain")[0]){
+    document.getElementsByClassName("clothWithStain")[0].style.pointerEvents = 'none';
+}
+if(document.getElementsByClassName("glassRod")[0]){
+    document.getElementsByClassName("glassRod")[0].style.pointerEvents = 'none';
+}
+if(document.getElementsByClassName("spoon2")[0]){
+    document.getElementsByClassName("spoon2")[0].style.pointerEvents = 'none';
+}
+if(document.getElementsByClassName("detergent2")[0]){
+    document.getElementsByClassName("detergent2")[0].style.pointerEvents = 'none';
+}
+if(document.getElementsByClassName("spoon3")[0]){
+    document.getElementsByClassName("spoon3")[0].style.pointerEvents = 'none';
+}
+if(document.getElementsByClassName("detergent3")[0]){
+    document.getElementsByClassName("detergent3")[0].style.pointerEvents = 'none';
+}
```

Comment:
Please create an array of all the element ids and add the pointer styling in a forEach loop